pgcli/main.py

```diff
@@ -469,6 +469,9 @@ def cli(database, user, host, port, prompt_passwd, never_prompt, dbname,
         print('Version:', __version__)
         sys.exit(0)
 
+    if not os.path.exists(os.path.expanduser('~/.config/pgcli')):
+        os.mkdir(os.path.expanduser('~/.config/pgcli'))
```

Comment:
Awesome catch. I had a `.config` folder, so it makes sense that it didn't fail locally. Thank you!